app/renderer/components/Inspector/Inspector.js

```diff
@@ -38,6 +39,19 @@ const MIN_WIDTH = 1080;
 const MIN_HEIGHT = 570;
 const MAX_SCREENSHOT_WIDTH = 500;
 
+function downloadXML (sourceXML) {
+  let element = document.createElement('a');
+  element.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(sourceXML));
```

Comment:
I believe there is a dedicated mime type for xml

Reply:
Thanks for the heads up! Making those changes right now.